modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIEndpointRelevantObjectEntryModelListener.java

```diff
@@ -198,7 +198,7 @@ else if (Objects.equals(method, Http.Method.POST)) {
 				filterString, apiEndpointObjectDefinition);
 
 			List<Map<String, Serializable>> valuesList =
-				_objectEntryLocalService.getValuesList(
+				_objectEntryLocalService.getPrimaryKeyList(
```

Comment:
What is the purpose of having a list of Maps and each map will return only one item with the same key always? Would it make sense to replace it with a list of ids?

Reply:
It was to maintain the same structure as before, but I agree it is not necessary and it is confusing. I will change it.